branding/common

```diff
@@ -9,3 +9,12 @@ export PORT=8080
 
 export MSI_PRODUCTCODE=415933d8-4104-47c3-aee9-66b31de07a57
 export OSX_IDPREFIX=org.jenkins-ci
+export AUTHOR="Kohsuke Kawaguchi <kk@kohsuke.org>"
+export LICENSE="MIT/X License, GPL/CDDL, ASL2"
+export HOMEPAGE=http://jenkins-ci.org/
+export CHANGELOG_PAGE=http://jenkins-ci.org/changelog
+
+export DESCRIPTION_FILE=`readlink -f description-file`
```

Comment:
IIUC, we need something more than this is to make sure that it picks up `description-file` that's in the branding directory, not on random `pwd`